etl/steps/data/grapher/regions/latest/regions.py

```diff
@@ -0,0 +1,305 @@
+"""
+
+import re
+
+from owid.catalog import Dataset, Table
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+# Legacy slugs for countries that have changed their name.
+LEGACY_SLUGS = {
+    "CZE": "czech-republic",
+    "MKD": "macedonia",
+    "SWZ": "swaziland",
+    "TLS": "timor",
+}
+
+# Countries that are mappable in Grapher.
+MAPPABLE_COUNTRIES = [
+    "AFG",
+    "AGO",
+    "ALB",
+    "AND",
+    "ARE",
+    "ARG",
+    "ARM",
+    "ATF",
+    "ATG",
+    "AUS",
+    "AUT",
+    "AZE",
+    "BDI",
+    "BEL",
+    "BEN",
+    "BFA",
+    "BGD",
+    "BGR",
+    "BHR",
+    "BHS",
+    "BIH",
+    "BLR",
+    "BLZ",
+    "BOL",
+    "BRA",
+    "BRB",
+    "BRN",
+    "BTN",
+    "BWA",
+    "CAF",
+    "CAN",
+    "CHE",
+    "CHL",
+    "CHN",
+    "CIV",
+    "CMR",
+    "COD",
+    "COG",
+    "COL",
+    "COM",
+    "CPV",
+    "CRI",
+    "CUB",
+    "CYP",
+    "CZE",
+    "DEU",
+    "DJI",
+    "DMA",
+    "DNK",
+    "DOM",
+    "DZA",
+    "ECU",
+    "EGY",
+    "ERI",
+    "ESH",
+    "ESP",
+    "EST",
+    "ETH",
+    "FIN",
+    "FJI",
+    "FRA",
+    "FSM",
+    "GAB",
+    "GBR",
+    "GEO",
+    "GHA",
+    "GIN",
+    "GMB",
+    "GNB",
+    "GNQ",
+    "GRC",
+    "GRD",
+    "GRL",
+    "GTM",
+    "GUF",
+    "GUY",
+    "HND",
+    "HRV",
+    "HTI",
+    "HUN",
+    "IDN",
+    "IND",
+    "IRL",
+    "IRN",
+    "IRQ",
+    "ISL",
+    "ISR",
+    "ITA",
+    "JAM",
+    "JOR",
+    "JPN",
+    "KAZ",
+    "KEN",
+    "KGZ",
+    "KHM",
+    "KIR",
+    "KNA",
+    "KOR",
+    "KWT",
+    "LAO",
+    "LBN",
+    "LBR",
+    "LBY",
+    "LCA",
+    "LIE",
+    "LKA",
+    "LSO",
+    "LTU",
+    "LUX",
+    "LVA",
+    "MAR",
+    "MCO",
+    "MDA",
+    "MDG",
+    "MDV",
+    "MEX",
+    "MHL",
+    "MKD",
+    "MLI",
+    "MLT",
+    "MMR",
+    "MNE",
+    "MNG",
+    "MOZ",
+    "MRT",
+    "MUS",
+    "MWI",
+    "MYS",
+    "NAM",
+    "NCL",
+    "NER",
+    "NGA",
+    "NIC",
+    "NLD",
+    "NOR",
+    "NPL",
+    "NRU",
+    "NZL",
+    "OMN",
+    "OWID_KOS",
+    "PAK",
+    "PAN",
+    "PER",
+    "PHL",
+    "PLW",
+    "PNG",
+    "POL",
+    "PRI",
+    "PRK",
+    "PRT",
+    "PRY",
+    "PSE",
+    "QAT",
+    "ROU",
+    "RUS",
+    "RWA",
+    "SAU",
+    "SDN",
+    "SEN",
+    "SGP",
+    "SLB",
+    "SLE",
+    "SLV",
+    "SMR",
+    "SOM",
+    "SRB",
+    "SSD",
+    "STP",
+    "SUR",
+    "SVK",
+    "SVN",
+    "SWE",
+    "SWZ",
+    "SYC",
+    "SYR",
+    "TCD",
+    "TGO",
+    "THA",
+    "TJK",
+    "TKM",
+    "TLS",
+    "TON",
+    "TTO",
+    "TUN",
+    "TUR",
+    "TUV",
+    "TWN",
+    "TZA",
+    "UGA",
+    "UKR",
+    "URY",
+    "USA",
+    "UZB",
+    "VCT",
+    "VEN",
+    "VNM",
+    "VUT",
+    "WSM",
+    "YEM",
+    "ZAF",
+    "ZMB",
+    "ZWE",
+]
+
+# Countries with no country page in Grapher.
```

Comment:
I think this list is incomplete. For example, "Abkhazia" doesn't have a country page either. Maybe it's better to define:
```
# All mappable countries should have a country page, and there are some that cannot be mapped but should also have a page.
HAS_COUNTRY_PAGE = MAPPABLE_COUNTRIES + [...]
```
And if there's any mappable country that shouldn't have a page, we could also remove it from the list here.
(This is my intuition, we would need to check)

Reply:
@samizdatco do you know if there's any relationship between mappable and listed countries? (like that `All mappable countries should have a country page`). If yes then I agree with Pablo and can explicitly whitelist them instead of blacklisting.

Reply:
(Implemented your suggestions @samizdatco)

This is crosstab of mappability and whether it is unlisted.
```
is_unlisted  False  True
is_mappable
False           62     21
True           200      2
```

Unfortunately "All mappable countries should have a country page" doesn't hold because Western Sahara (ESH) and French Southern Territories (ATF) are mappable, but doesn't have country profiles.

I guess I could do
```
# All mappable countries should have a country page, and there are some that cannot be mapped but should also have a page.
HAS_COUNTRY_PAGE = MAPPABLE_COUNTRIES + NON_MAPPABLE_WITH_COUNTRY_PAGE - MAPPABLE_WITHOUT_COUNTRY_PAGE
```
but that's not much simpler as the original blacklisting approach. I'd probably keep it as it is now.

Reply:
Thanks @Marigold. For some reason I'm not seeing the OWID_ZAN change reflected in the csv yet. Also, if we're keeping the blocklist approach, can you add PS_GZA and SXM to the non-country-page group?

Reply:
My bad, forgot to push this to staging. It should be there in a couple minutes.